app/javascript/src/Stats/buyer/stats_application.jsx

```diff
@@ -7,6 +7,13 @@ import {StatsMethodsTable} from 'Stats/lib/methods_table'
 import {StatsApplicationMetricsSource} from 'Stats/lib/application_metrics_source'
 import {StatsCSVLink} from 'Stats/lib/csv_link'
 import {Stats} from 'Stats/lib/stats'
+import {StatsStore} from 'Stats/lib/store'
+import $ from 'jquery'
+
+function getStoredApplicationId () {
+  let storedState = new StatsStore(window).getStateFromURL()
```

Comment:
Suggestion: rename `storedState` to `state`. The more words the more complex to read.

Reply:
it's good to be explicit here, that's stored or it's undfined. const could do.